Corrected author formatting for some papers, added a category and papers.

kqwang opened the pull request:
Hi. I corrected author formatting for some papers, and added a new category and papers.

guillermogb reviewed (changes requested):
Thanks. 
I do not think that adding "and" in the two-author entries adds much value. Hence, I do not recommend it. Could you restore those entries? Then we can focus on other matters. For example, I see that Mueggler17ijrr now has two primary entries (two anchors with the same identifier on the same page), and makes it difficult to maintain. Please use a primary entry and secondary entries that link to the primary one.

kqwang commented:
Agree. As in the two new commits added, I have restored all entries with two authors I changed before. In addition, I changed the former of the two primary entries of "Mueggler17ijrr" to a secondary entry.

guillermogb commented on README.md:
```diff
@@ -452,7 +453,7 @@ British Machine Vision Conf. (BMVC), 2020. [PDF](https://www.bmvc2020-conference
 *[e-ACJ: Accurate Junction Extraction For Event Cameras](https://arxiv.org/pdf/2101.11251)*,  
 arXiv, 2021.
-- <a name="Dong21ICMVA"></a>Dong, Y., Zhang, T.,  
+- <a name="Dong21ICMVA"></a>Dong, Y. and Zhang, T.,  
```
no need for "and"

kqwang replied:
Done. This is an original issue, existed before my first pull request.

guillermogb replied:
thanks. Somehow it was highlighted in the rich diff

guillermogb commented on README.md:
```diff
@@ -452,7 +453,7 @@ British Machine Vision Conf. (BMVC), 2020. [PDF](https://www.bmvc2020-conference
 *[e-ACJ: Accurate Junction Extraction For Event Cameras](https://arxiv.org/pdf/2101.11251)*,  
 arXiv, 2021.
-- <a name="Dong21ICMVA"></a>Dong, Y., Zhang, T.,  
+- <a name="Dong21ICMVA"></a>Dong, Y. and Zhang, T.,  
```
we use lowercase for publication venue: Dong21icmva

kqwang replied:
Done. This is an original issue, existed before my first pull request.

guillermogb replied:
thanks

guillermogb commented on README.md:
```diff
@@ -2386,6 +2389,12 @@ Optics Letters, 2021.
 Optics Express, 2022.
 
+<a name="coherence_scanning_interferometry"></a>
+## Coherence Scanning Interferometry
```
Since this subsection has only 1 entry and the previous one has also only 4, would you consider merging them, potentially together with other entries into something like "Optics Applications" ? @kqwang

kqwang replied:
I agree. I merged all new paper I added before into one part as "Optical Applications".

guillermogb commented on README.md:
```diff
@@ -2630,7 +2640,7 @@ IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2021. [Pro
 *[Dynamic Event Camera Calibration](https://arxiv.org/abs/2107.06749)*,  
 IEEE/RSJ Int. Conf. Intelligent Robots and Systems (IROS), 2021. [Video](https://www.bilibili.com/video/BV1ey4y1j7Ke?share_source=copy_web), [Code](https://github.com/MobilePerceptionLab/EventCameraCalibration).
-- <a name="BaoArxiv23"></a>Bao, Y., Sun, L., Ma, Y., Gu, D., Wang, K.,  
+- <a name="Bao23arxiv"></a>Bao, Y., Sun, L., Ma, Y., Gu, D., Wang, K.,  
```
thanks for removing the two primary entries. I overlooked it.

guillermogb reviewed (approved):
thank you!

kqwang commented:
I will keep an eye on this great "Resources" that you created and inject more optical applications. By the way, I am building a similar "Resources" for my research field, and I was more inspired by this Resources you made. Expected to be open source within a month.

guillermogb commented:
Thanks, @kqwang . You seem to be meticulous. We greatly appreciate your contributions to this repository. I was not aware of several of the entries you wrote. 
BTW, I also have some updates in this [spreadsheet](https://docs.google.com/spreadsheets/d/1_OBbSz10CkxXNDHQd-Mn_ui3OmymMFvm-lW316uvxy8)

kqwang commented:
Thanks for your sharing.
